magento/pwa-studio
# [feature]: Update manifest with new Venia theme/background color

## Description

Replacement of theme_color and review of others in manifest so that it appears correctly in app. Currently, color should be HEX #1F39FF 

## Related Issue





Closes #2802.

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Access to site.
2. Click install app on browser (with desktop) or Add to home screen (with mobile version)
3. Check the background theme color as issue reported.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
